Runigram.java

```diff
@@ -35,14 +35,18 @@ public static Color[][] read(String fileName) {
 		int numCols = in.readInt();
 		int numRows = in.readInt();
 		in.readInt();
+		int red, green, blue;

```

Comment:
no need to declare variables outside of it use. See Java Style Guide in the moodle